cg/apps/housekeeper/hk.py

```diff
@@ -55,7 +55,7 @@ def create_new_bundle_and_version(self, name: str) -> Bundle:
         return new_bundle
 
     def set_to_archive(self, file: File, value: bool) -> None:
-        """Sets the 'to_archive' field of a file."""
+        """Sets the 'is_archiving_request' field of a file."""
```

Comment:
This doc string is confusing to me, maybe it can be removed?

Reply:
A name alternative: `mark_file_as_ready_to_archive`

Reply:
This change was made accidentally, due to a refactoring 😅  Removing